R/eval_forecasts.R

```diff
@@ -109,6 +109,19 @@
 #' @param merge_by character vector with column names that `forecasts` and
 #' `truth_data` should be merged on. Default is `NULL` and merge will be
 #' attempted automatically.
+#' @param compute_relative_skill logical, whether or not to compute realitve
```

Comment:
*relative (I'd suggest adding automated package spell checking using the `spelling` package (see `devtools`).

Reply:
good suggestion, will do :)